static/i18n/en/app.json

```
@@ -2339,7 +2339,8 @@
   },
   "informationCenter": {
     "tooltip": "Information center",
-    "ongoingIncidentsTooltip": "There are {{incidentCount}} ongoing incidents",
+    "ongoingIncidentsTooltip": "There is {{count}} ongoing incidents",
```

Since it's a known 1, remove the {{count}} altogether 🤔, or at least remove the trailing "s"
```suggestion
    "ongoingIncidentsTooltip": "There is 1 ongoing incident",
```

---

i'll remove the S indeed but we never know we can have 0 incidents :laughing: